src/interest-groups/constants.js

```diff
@@ -0,0 +1,14 @@
+const hoursPerDay = 24;
+const minutesPerHour = 60;
+const secondsPerMinute = 60;
+const millisecondsPerSecond = 1000;
+export const MS_PER_DAY = hoursPerDay * minutesPerHour * secondsPerMinute * millisecondsPerSecond;
```

Comment:
you should make this a static `86400000` to avoid calculating on every page load.

Reply:
commit: 1020be68a09c77d23306b8e021596d4a4ff4cb06